docs/source/user-guide/vision/quickstarts/plot_detection_tutorial.py

```diff
@@ -31,29 +31,8 @@
 #%%
 # Defining the data and model
```

Comment:
I think this makes it look like we only support pytorch (because people sometimes only look at the header). Instead, added a comment about it in text (like Bressler did). WDYT?

Reply:
also works :)